.github/workflows/linting.yml

```diff
@@ -1,28 +1,20 @@
 ---
 name: Linting
 
-# yamllint disable-line rule:truthy
 on:
   push:
     branches:
       - main
       - renovate/**
   pull_request:
-    types:
-      - opened
-      - ready_for_review
-      - reopened
-      - synchronize
```

Comment:
was there an issue with linting not running when it should?

Reply:
This was thinking about reducing CI usage after MIRSG-locked-out-of-GitHub-Actions gate, but this is a private repo, so not worth it